roles/common/vars/main.yml

```diff
@@ -2,9 +2,6 @@
 homebrew_tap:
   - homebrew/services
   - buo/cask-upgrade
-  - homebrew/cask-fonts
-  - homebrew/cask-versions
-  - akollade/tap
```

Comment:
I also deleted our tap, we don't use it anymore since mailpit is now integrated in brew core